apps/node/index.ts

```diff
@@ -26,17 +27,15 @@ const prompt = `Press <enter> to run the next test...`;
 const promptToContinue = () =>
   new Promise((resolve) => cli.question(prompt, (_answer) => resolve()));
 
-// This needs to be more sophisticated to work on Linux and Windows as well.
-const openPdf = (path: string) => {
+// This needs to be more sophisticated to work on Linux as well.
+const openPdf = (path: string, reader: string) => {
   if (process.platform === 'darwin') {
-    // TODO: Make this a CLI argument
-    execSync(`open -a "Preview" ${path}`);
-    // execSync(`open -a "Adobe Acrobat" ${path}`);
-    // execSync(`open -a "Foxit Reader" ${path}`);
-    // execSync(`open -a "Google Chrome" ${path}`);
-    // execSync(`open -a "Firefox" ${path}`);
+    execSync(`open -a "${reader}" ${path}`);
+  } else if (process.platform === 'win32') {
+    // Opens with the default PDF Reader, has room for improvment
+    execSync(`start ${path}`);
   } else {
-    const msg1 = `Note: Automatically opening PDFs currently only works on Macs. If you're using a Windows or Linux machine, please consider contributing to expand support for this feature`;
+    const msg1 = `Note: Automatically opening PDFs currently only works on Macs and Windows. If you're using a Linux machine, please consider contributing to expand support for this feature`;
```

Comment:
Sorry, this was the code I wanted to comment on..

Running that many integration tests to see if it works, I created a little CLI arg parser, to detect a `--reader` flag on MacOSX, if missing it defaults to Preview and for other programs, it is needed to type for example `yarn apps:node --reader="Adobe Acrobat.app`.

As usual feedback is welcome.